File: Exercicio-07.java
import java.util.Scanner;
import java.text.DecimalFormat; 

public class Main
{
  public static void main (String[]args)
  {
    Scanner sc = new Scanner (System.in);
    DecimalFormat df = new DecimalFormat("#,###.00");
    double n1, n2, avg;
    int p1, p2;

    System.out.printf("Digite a primeira nota: ");
    n1 = sc.nextDouble();
  
    System.out.printf("Digite o peso referente a essa nota: ");
    p1 = sc.nextInt();
      
    System.out.printf("Digite a segunda nota: ");
    n2 = sc.nextDouble();
      
    System.out.printf("Digite o peso referente a essa nota: ");
    p2 = sc.nextInt();
    
    
    avg = ((n1 * p1) + (n2 * p2)) / (p1 + p2);
    
    System.out.printf("A média ponderada é: " + df.format(avg));
  }
}

File: Exercicio-07.c
#include <stdio.h>

int main(void) {

  float n1, n2, avg;
  int p1, p2;

  printf("Digite a primeira nota: ");
  scanf("%f", &n1);
  
  printf("Digite o peso referente a essa nota: ");
  scanf("%d", &p1);
  
  printf("Digite a segunda nota: ");
  scanf("%f", &n2);
  
  printf("Digite o peso referente a essa nota: ");
  scanf("%d", &p2);
  
  avg = ((n1 * p1) + (n2 * p2)) / (p1 + p2);
  
  printf("A média ponderada é: %.2f", avg);
  
  return 0;
}
